src/main/java/epam/task/gymbootdb/controller/TraineeController.java

```diff
@@ -0,0 +1,118 @@
+package epam.task.gymbootdb.controller;
+
+import epam.task.gymbootdb.dto.TraineeDto;
+import epam.task.gymbootdb.dto.TrainerDto;
+import epam.task.gymbootdb.dto.UserCredentials;
+import epam.task.gymbootdb.service.TraineeService;
+
+import io.swagger.v3.oas.annotations.Operation;
+import io.swagger.v3.oas.annotations.responses.ApiResponse;
+import io.swagger.v3.oas.annotations.responses.ApiResponses;
+import jakarta.validation.Valid;
+
+import lombok.RequiredArgsConstructor;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+@RequestMapping("/trainees")
+@RequiredArgsConstructor
+public class TraineeController {
+
+    private final TraineeService traineeService;
+
+    @Operation(summary = "Get trainee by ID",
```

Comment:
Let's create an interface for each controller and place swagger annotation there